### ISSUE TITLE
Book Detail Page

### ISSUE THREAD

**Issue by louisweiser** [louisweiser/project_capstone#7]

**Comment by rabe200** [comment]
lgtm so far ... just the value proposition is unclear to me. 

should there happen anything on the page displayed if clicking the cover image?
or does this sentence relate to a previous page that is leading to this page here? 
-> "The view should be accessible by clicking on the respective cover."

**Comment by louisweiser** [comment]
yes this sentence relate to a previous page that is leading to this page here. I adjust the description for better clarity. Thank You
